Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -894,6 +895,17 @@ void node_implementation_init(nmos::node_model& model, slog::base_gate& gate)
         auto channelmapping_output = nmos::make_channelmapping_output(id, name, description, source_id, channel_labels, routable_inputs);
         if (!insert_resource_after(delay_millis, model.channelmapping_resources, std::move(channelmapping_output), gate)) throw node_implementation_init_exception();
     }
+
+    // example root block
+    auto root_block = nmos::make_root_block();
+    // example device manager
+    auto device_manager = nmos::make_device_manager(2, root_block, model.settings);
+    if (!insert_resource_after(delay_millis, model.control_protocol_resources, std::move(device_manager), gate)) throw node_implementation_init_exception();
+    // example class manager
+    auto class_manager = nmos::make_class_manager(3, root_block);
```

Comment:
Same as for line 902?

Reply:
This is the `oid` for the device manager instance, which can be between `2-65535`, as the root block can only be `1`. and `2` has been used for the device manager instance.